text/0000-generalized-type-ascription.md

```diff
@@ -1526,6 +1526,55 @@ Furthermore, it was also noted on the same PR by @nikomatsakis that:
 
 Thus, we believe it is reasonable to expedite the schedule.
 
+## Precedence of the operator
+
+As explained prior, we change the precedence of `:` when in an expression
+context such that `x : T.foo()` is interpreted as `(x : T).foo()`.
+This precedence change allow users to write readable code when they
+have several method calls by using line-separation such as with:
+
+```rust
+let x = (0..10)
+    .map(some_computation)
+    .collect() : Foo
+    .unwrap()
+    .map(other_computation) : Bar
+    .into() : Baz;
+```
+
+However, if you write this on a single line, or simply consider `x : T.foo()`
+a user might parse this as `x : (T.foo())` instead.
+While Rust does not have any "type level methods", and wherefore this parse
```

Comment:
"wherefore" isn't right here ("therefore" is closer, but still awkward). Maybe rephrase to something like: "While at this stage Rust does not support type-level methods (meaning this parse currently makes no sense), a user may nonetheless make this mistake."